packages/client/lib/Chain.js

```diff
@@ -187,6 +187,30 @@ export default class Chain {
     return transaction
   }
 
+  /**
+   * Create & sign a sweep transaction
+   * @param {!string} externalAddress - External Change address.
+   * @param {string[]} outputs - to, value.
+   * @param {number} [fee] - Fee price in native unit (e.g. sat/b, wei)
+   * @param {string[]} fixedInputs - txid, vout, address, value, satoshis, derivationPath
+   * @return {Promise<string>} Resolves with a signed transaction object.
+   */
+  async buildSweepTransaction (externalAddress, fee, outputs, fixedInputs) {
+    return this.client.getMethod('buildSweepTransaction')(externalAddress, fee, outputs, fixedInputs)
+  }
+
+  /**
+   * Create, sign & broadcast a sweep transaction.
+   * @param {!string} externalAddress - External Change address.
+   * @param {string[]} outputs - to, value.
+   * @param {number} [fee] - Fee price in native unit (e.g. sat/b, wei)
+   * @param {string[]} fixedInputs - txid, vout, address, value, satoshis, derivationPath
+   * @return {Promise<Transaction>} Resolves with a signed transaction.
+   */
+  async sendSweepTransaction (externalAddress, fee, outputs, fixedInputs) {
```

Comment:
On the client, signatures should not include anything that we are not able to abstract across chains. Should be `sendSweepTransaction (address, fee)`

Reply:
Good point, updated in https://github.com/liquality/chainabstractionlayer/pull/358/commits/3100c572312cdd9a3510e223d7ec76ca921a060e